chia/_tests/timelord/test_new_peak.py

```diff
@@ -115,14 +116,11 @@ async def test_timelord_new_peak_unfinished_not_orphaned(
             await _validate_and_add_block(b1, block_2)
 
             block_record = b1.block_record(block_2.header_hash)
-            sub_slot_iters, difficulty = get_next_sub_slot_iters_and_difficulty(
-                bt.constants, len(block_1.finished_sub_slots) > 0, b1.block_record(block_1.prev_header_hash), b1
-            )
 
             timelord_unf_block = timelord_protocol.NewUnfinishedBlockTimelord(
                 block_1.reward_chain_block.get_unfinished(),
-                difficulty,
-                sub_slot_iters,
+                uint64(block_record.weight - default_1000_blocks[-1].weight),
```

Comment:
removed the redundant iterations/difficulty calculation

Reply:
Does this still need to get committed? @almogdepaz @arvidn